app/sagas/deepLinking.js

```
@@ -1,5 +1,5 @@
 import { InteractionManager } from 'react-native';
-import RNCallKeep from 'react-native-callkeep';
+import { voipNative } from '../lib/services/voip/VoipNative';
```

ESLint

[error] `../lib/services/voip/VoipNative` import should occur after import of `redux-saga/effects`

(import/order)